zilencer/lib/stripe.py

```diff
@@ -178,3 +179,93 @@ def do_subscribe_customer_to_plan(stripe_customer: stripe.Customer, stripe_plan_
                 event_time=timestamp_to_datetime(stripe_subscription.created),
                 requires_billing_update=True,
                 extra_data=ujson.dumps({'quantity': current_seat_count}))
+
+## Process RealmAuditLog
+
+class BillingLogProcessingException(Exception):
+    pass
+
+def is_customer_agnostic_error(error: Exception):
+    # TODO
+    return False
+
+def get_next_log_entry(processor: RealmAuditLogProcessor) -> Optional[RealmAuditLog]:
+    if processor.realm is None:
+        realms_with_processors = RealmAuditLog.objects.values_list('realm', flat=True)
+        query = RealmAuditLog.objects.exclude(realm__in=realms_with_processors)
+    else:
+        query = RealmAuditLog.objects.filter(realm=processor.realm)
+        global_processor = RealmAuditLog.objects.get(realm=None)
```

Comment:
Is there only ever going to be a single entry in `RealmAuditLog` with `realm=None`?  Or is this meant to be a `RealmAuditLogProcessor`?